code/UMLEditor.py

```diff
@@ -14,6 +14,29 @@
 import UMLClass
 import UMLRelationship
 
+##########################################################################
+# Constants 
+
+COMMANDS = {
+    "help" : [
+        {
+            "usage" : "help", 
+            "desc" : "lists all possible commands"
+        },
+        {
+            "usage" : "help <command_name>", 
+            "desc" : "lists the usage for <command_name>"
+        }
+    ],
+    "create_class" : [
+        {
+            "usage" : "create_class <class_name>",
+            "desc" : "creates a class with <class_name> as the name"
+        }
+    ]
+}
+
```

Comment:
As a separate task, we can go through and add in all the commands, usage statements, and descriptions. This was just a sample to test that the function works